web/router.go

```diff
@@ -56,6 +59,55 @@ func Router(app *services.ChainlinkApplication) *gin.Engine {
 	return engine
 }
 
+// secureOptions configure security options for the secure middleware, mostly
+// for TLS redirection
+func secureOptions(config store.Config) secure.Options {
+	options := secure.Options{
+		IsDevelopment: config.Dev,
+		SSLRedirect:   config.TLSPort != 0,
+	}
+
+	if config.TLSHost != "" {
+		if config.TLSPort != 0 && config.TLSPort != 443 {
```

Comment:
Optional nitpick: Why even divide `TLSHost` from `TLSPort` if the underlying `SSLHost` takes both as a string? I say surface `SSLHost` to take a string with both host and port and remove these `if`s.

```golang
if config.SSLHost != "" {
  options.SSLHost = config.SSLHost
}
```

Has less config options as well which I see as a win.